includes/api/auth.php

```diff
@@ -114,6 +114,18 @@ public function verifyResourceRequest(iaApiRequest $request)
 
 	public function getAccessTokenData(iaApiRequest $request)
 	{
+		$permanentKey = $this->iaCore->get('api_token');
+		if ($permanentKey and $permanentKey == $request->getServer('HTTP_X_AUTH_TOKEN'))
```

Comment:
@nkanaev please use && instead of *and*, || instead *of* or, etc in control statements. Not a critical one, just a recommendation. (yep, not described in standards)